save-api-cli/src/main/kotlin/com/saveourtool/save/apicli/ArgsParsing.kt

```diff
@@ -80,9 +80,10 @@ fun parseArguments(args: Array<String>): CliArguments? {
 
     parser.parse(args)
 
+    // FIXME
     val authorization = oauth2Source?.let {
```

Comment:
API should not use OAUTH 2

Reply:
what do you mean? its just `basic` or `github` in most of the cases

could be just renamed to `source`

Reply:
`github` is oauth2. We can't use it in API. Only basic auth (token)

Reply:
agree